lecture/models.py

```
@@ -44,26 +44,27 @@
         (PENDING, 'Pending'),
         (PROCESSING, 'Processing'),
         (COMPLETED, 'Completed'),
     )
-    lecture=models.ForeignKey('Lecture',on_delete=models.CASCADE,related_name='videos')
-    video_file=models.FileField(upload_to='lecture_videos/')
-    video_title=models.TextField(null=True, blank=True)
-    thumbnail = models.ImageField(upload_to="thumbnails",null=True,blank=True)
-    hls = models.CharField(max_length=500,blank=True,null=True)
+    lecture = models.ForeignKey('Lecture', on_delete=models.CASCADE, related_name='videos')
+    video_file = models.FileField(upload_to='lecture_videos/')
+    video_title = models.TextField(null=True, blank=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `TextField`

(DJ001)